Sketch.java

```diff
@@ -8,28 +8,67 @@
    */
   public void settings() {
 	// put your size call here
-    size(400, 400);
+    size(800, 800);
   }
 
   /** 
    * Called once at the beginning of execution.  Add initial set up
    * values here i.e background, stroke, fill etc.
    */
   public void setup() {
-    background(210, 255, 173);
+    background(34, 206, 245);
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
 	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
+	// ground color (foreground)
+    stroke(74, 50, 1);
+    fill(74, 50, 1);
+    rect(0, 650, 800, 200);
+    
+  // lawn (foreground)
+    stroke(0);
+    fill(22, 181, 72);
+    rect(150, 650, 550, 500);
+
+  // pathway (foreground)
+    stroke(0);
+    fill(130, 148, 148);
+    rect(390, 650, 80, 200);    
+    
+  // house shape
+    stroke(0);
+    fill(247, 186, 62);
+    rect(250, 300, 350, 350);
+
```

Comment:
line up your comments to be in line with the code they refer to, don't outdent your comments as you are doing since it makes the code less readable.